lms/views/lti/basic_launch.py

```diff
@@ -277,16 +277,10 @@ def _configure_js_to_show_document(self, document_url, assignment):
             # nb. Canvas does not currently use/support any functionality from the
             # instructor toolbar. For grading it uses SpeedGrader and we don't
             # support editing assignments.
-
-            enable_editing = self.context.application_instance.settings.get(
-                "hypothesis", "edit_assignments_enabled", default=False
+            self.context.js_config.enable_instructor_toolbar(
```

Comment:
Always show the instructor bar for instructors. Pass the right value for `enable_grading`.